types.bal

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+public type Attribute record {
+    string name;
+    string value;
+};
+
+public type CreateDomainResponse record {
+    ResponseMetadata responseMetadata;
+};
+
+public type DeleteDomainResponse record {
+    ResponseMetadata responseMetadata;
+};
+
+public type ListDomainsResponse record {
+    ListDomainsResult listDomainsResult;
+    ResponseMetadata responseMetadata;
+};
+
+public type ListDomainsResult record {
+    string domainNames;
+    string nextToken;
+};
+
+public type SelectResponse record {
+    SelectResult selectResult;
+    ResponseMetadata responseMetadata;
+};
+
+public type SelectResult record {
+    string items;
```

Comment:
For public types, we need to add doc comments

Reply:
Added doc comments